bin/slimerjs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+/**
+ * Script that will execute firefox.  stdio are
+ * forwarded to and from the child process.
+ *
+ * The following is for an ugly hack to avoid a problem where the installer
```

Comment:
I don't think we need this hack because the binpath is not searched for, it's known already.